src/docfx/build/xref/ExternalXrefMapLoader.cs

```diff
@@ -133,19 +169,32 @@ private static (List<(string uid, long start, long end)>, string?) GetXrefSpecPo
                                 throw Errors.JsonSchema.UnexpectedType(new SourceInfo<string>(filePath), "string", reader.TokenType).ToException();
                             }
                         }
+                        if (reader.ValueTextEquals(s_referencesBytes) || reader.ValueTextEquals(s_externalXrefsBytes))
```

Comment:
check stack depth to ensure `externalXrefs` is the top level property. The `ValueTextEquals` in line 172 seem to duplicate with line 174.

Reply:
OK.